### PR TITLE
Add `@author` to forbidden tag list

### LINKED ISSUE #167: @author MUST NOT be used.
### Rule

@author MUST NOT be used

### Reason

As per magento DocBlock document (https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#documentation-space)

Structure of documentation space
@author ,@category, @package, and @subpackage MUST NOT be used. Documentation is organized with the use of namespaces.

### Implementation

Need to add @author in $forbiddenTags

### PR DESCRIPTION
**Pull request by fredden**
Ref https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#documentation-space
Fixes #167

### CODE REVIEW

**Review by serzhik — DISMISSED** [review]

**Comment by dfelton** [comment]
A spelling typo exists in these changes proposed.

>  * @author is not ment to be used

`ment` should be `meant`, the past tense form of the word `mean`.

File: Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.1.inc [hunk 1]
```diff
@@ -60,7 +60,7 @@ class EmptyHandler
  * @api is ok here
  * @deprecated can be used in this context
  * @see is ok here
- * @author is actually ok
+ * @author is not ment to be used
```
Comment (dfelton) [inline Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.1.inc:63]:
```suggestion
 * @author is not meant to be used
```

File: Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.2.inc [hunk 2]
```diff
@@ -60,7 +60,7 @@ interface EmptyHandler
  * @api is ok here
  * @deprecated can be used in this context
  * @see is ok here
- * @author is actually ok
+ * @author is not ment to be used
```
Comment (dfelton) [inline Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.2.inc:63]:
```suggestion
 * @author is not meant to be used
```

**Review by dfelton — CHANGES_REQUESTED** [review]
Spelling correction

File: Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.1.inc (hunk 1 above)
Comment (fredden) [reply Magento2/Tests/Commenting/ClassAndInterfacePHPDocFormattingUnitTest.1.inc]: Thanks for spotting this. I copied the text from two lines below and didn't spot this mistake in the original. I've fix this and the original now.

**Review by dfelton — APPROVED** [review]

**Review by sidolov — APPROVED** [review]

**Comment by sidolov** [comment]
@magento import pr to magento-commerce/magento-coding-standard repository

**Comment by m2-github-services** [comment]
@sidolov the Pull Request is successfully imported.

**Pull request merged**